dev-libs/pigpio/files/pigpio-66-makefile.patch

```diff
@@ -0,0 +1,117 @@
+--- a/Makefile	2018-03-20 19:57:38.812609685 +0100
++++ b/Makefile	2018-03-20 20:26:58.823455489 +0100
+@@ -12,31 +12,31 @@
+ 
+ CFLAGS	+= -O3 -Wall -pthread
+ 
+-LIB1     = libpigpio.so
```

Comment:
Is this really a good idea? Unless you can reliably upstream this, you're creating a fake SOVERSION. If you really want to do this, then please use some indication that the SOVERSION is custom, e.g. by naming it `gentoo0`.

Reply:
I already requested SOVERSIONing upstream, so for the moment I think I'll just scrap all those changes and keep the libs without version. That should also solve the issues with the direct linking above.